tracer/src/Datadog.Trace/Debugger/Sink/ProbeStatusSink.cs

```diff
@@ -54,21 +54,21 @@ internal void AddBlocked(string probeId)
 
         internal void AddError(string probeId, Exception e)
         {
-            AddProbeStatus(probeId, Status.ERROR, e);
+            AddProbeStatus(probeId, Status.ERROR, exception: e);
         }
 
-        public void AddProbeStatus(string probeId, Status status, Exception exception = null, string errorMessage = null)
+        public void AddProbeStatus(string probeId, Status status, int probeVersion = 0, Exception exception = null, string errorMessage = null)
         {
             var shouldSkip =
                 _diagnostics.TryGetValue(probeId, out var current) &&
-                !ShouldOverwrite(current.Message.DebuggerDiagnostics.Diagnostics.Status, status);
+                !ShouldOverwrite(current.Message.DebuggerDiagnostics.Diagnostics.Status, status, current.Message.DebuggerDiagnostics.Diagnostics.ProbeVersion, probeVersion);
```

Comment:
nit: suggest adding named params `oldVersion:` and `newVersion:` here for clarity

Reply:
Done in fba839d65ca67eeb3717c9280c24132a8c329d33.